ckanext/useraffiliation/actions.py

```diff
@@ -0,0 +1,109 @@
+import copy
+import datetime
+import logging
+from ckan import model, logic
+from ckan.plugins import toolkit
+
+log = logging.getLogger(__name__)
+
+
+def _get_user_obj(context):
+    if "user_obj" in context:
+        return context["user_obj"]
+    user = context.get("user")
+    m = context.get("model", model)
+    user_obj = m.User.get(user)
+    if not user_obj:
+        raise toolkit.ObjectNotFound("User not found")
+    return user_obj
```

Comment:
To get rid of ckan.model dependency, and also to improve readability? 

```suggestion
def _get_user_obj(context):

    if "user_obj" in context:
        user_obj = context["user_obj"]
    elif "model" in context and "user" in context:
        user_obj = context['model'].User.get(context['user'])
        
    if not user_obj:
        raise toolkit.ObjectNotFound("No user object could be found")
        
    return user_obj
```